back/engines/commercial/bulk_import_ideas/spec/services/extractors/base_extractor_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe BulkImportIdeas::Extractors::BaseExtractor do
+  let(:service) { described_class.new('en') }
+
+  let(:column_name) { 'A question name here' }
+
+  describe '#clean_string_value' do
+    it 'removes extra spaces around commas and colons' do
+      value = '  This is a test,   with some : extra spaces , and : colons  '
```

Comment:
Could you ever have some cases like this "This is a test ,with a comma"? Because I think that wouldn't get cleaned correctly.

Reply:
Guess there is a possibility, but it's mainly for cases where the text is delimited - we care less about it in the middle of text somewhere